changelog/unreleased/pr-23133.toml

```diff
@@ -0,0 +1,5 @@
+type = "f"
```

Comment:
This one is probably not needed.

From changelog docs:
```
File Location
Each changelog entry must be in a separate file. The file name should either be issue-<issue-number>.toml or pr-<pr-number>.toml. Only use the pr- name format if there is no related issue for a change in a pull request or if the issue is from a different repository.
```

Reply:
Heh, I think of myself being lazier than I actually am, I forgot I already created a changelog snippet :D